ios/FluentUI/Popup Menu/MSPopupMenuItem.swift

```diff
@@ -33,8 +33,10 @@ open class MSPopupMenuItem: NSObject {
     @objc public var isSelected: Bool = false
 
     @objc public let onSelected: (() -> Void)?
+    
+    @objc public var checkmarkVisibleOnSelect: Bool = true
```

Comment:
making this "public var" makes me think that the client can change this dynamically and it will change the behavior.... i think it might be better as "let"